packages/flutter/test/material/input_chip_test.dart

```diff
@@ -321,14 +321,14 @@ void main() {
     expectCheckmarkColor(find.byType(InputChip), Colors.white.withAlpha(0xde));
   });
 
-  testWidgets('Material3 - Input chip check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+  testWidgets('Material3 - Input chip disabled check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
       brightness: Brightness.dark,
     );
-
-    expectCheckmarkColor(find.byType(InputChip), Colors.white.withAlpha(0xde));
+    // Without color opacity. _RenderChip handles the 'disabled' state opacity internally
```

Comment:
```suggestion
```
We shouldn't document private class.

Expected results already matches the spec.

Reply:
Ok. I placed the note, because the M2 color has an opacity, whereas M3 doesn't, although the spec in M3 specifies an opacity.